src/python/install.sh

```diff
@@ -344,13 +344,20 @@ install_python() {
         check_packages python3 python3-doc python3-pip python3-venv python3-dev python3-tk
         INSTALL_PATH="/usr"
 
-        ln -s "${INSTALL_PATH}/bin/python3" "${INSTALL_PATH}/bin/python"
-        ln -s "${INSTALL_PATH}/bin/pydoc3" "${INSTALL_PATH}/bin/pydoc"
-        ln -s "${INSTALL_PATH}/bin/python3-config" "${INSTALL_PATH}/bin/python-config"
-
-        # Add the current symlink but point it to "/usr" since python is at /usr/bin/python
-        mkdir -p "${PYTHON_INSTALL_PATH}"
-        add_symlink
+        local current_bin_path="${CURRENT_PATH}/bin"
+        if [ "${OVERRIDE_DEFAULT_VERSION}" = "true" ]; then
+            rm -rf "${current_bin_path}"
+        fi
+        if [ ! -d "${current_bin_path}" ] ; then
+            mkdir -p "${current_bin_path}"
+            # Add an interpreter symlink but point it to "/usr" since python is at /usr/bin/python, add other alises
+            ln -s "${INSTALL_PATH}/bin/python3" "${current_bin_path}/python3"
```

Comment:
I believe you are hoping the `/usr/local/python/current` folder to be owned by `USERNAME:python` ?

<img width="943" alt="image" src="https://user-images.githubusercontent.com/24955913/201214497-ac5d2508-34c3-41e0-afe2-0fd6d68af1ca.png">

Then the condition needs to be changed 🤔 

https://github.com/devcontainers/features/blob/e9c8ba5cf598a65957d8f25cd9515e0db88e4da0/src/python/install.sh#L407

Reply:
Given this is the OS installed version of python, we shouldn't be messing around with privs TBH.  It was actually like that before.

Reply:
Updated